examples/graphene_apps/python_worker/compose/ov-subnet.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+# Define network and assign to static ip containers.
+# GSC tool/Graphene has an issue and is not able to resolve DNS name to IP.
```

Comment:
Is it because of proxy? Not clear why GSC tool has to resolve the DNS name to IP?  
10.5.0.6 is used in zmq communication right? What is role of GSC tool here?

Reply:
Please check The Graphene issue -> https://github.com/oscarlab/graphene/issues/1767